app/views/crime_applications/index.html.erb

```diff
@@ -21,7 +21,7 @@
   <table class="govuk-table app-dashboard-table">
     <thead class="govuk-table__head">
       <tr class="govuk-table__row">
-        <%= render "#{@review_status}_crime_applications_table_headers", sorting: @sorting %>
+        <%= render "#{@review_status}_crime_applications_table_headers", sorting: @search.sorting %>
```

Comment:
Where is this being set - is it from params? could someone use it to request a file they aren't supposed to have access to? 🤔

Reply:
set per action, not from params.

Reply:
Agreed to have separate views for open and closed rather than sharing index.